package/yast2.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri May 17 11:17:17 CEST 2019 - aschnell@suse.com
+
+- create log directory with control.xml and merged installation.xml
```

Comment:
I worry that it will need bsc, jira or fate number.

Reply:
Not for factory (according to the openSUSE rules). For SP2 we can assume that it gets accepted together with other changes that do have a number.